Mainichi Daily News.js

```diff
@@ -80,7 +80,7 @@ function doWeb(doc, url) {
 function scrape(doc, url) {
 	var item = new Zotero.Item("newspaperArticle");
 
-	item.title = ZU.xpathText(doc, '//header/h1');
+	item.title = text(doc, 'header>h1, .header-box h1');
```

Comment:
`text()` and `attr()` will not work in a number of settings (e.g. translation server), so you still need to define them (there's a long answer from dstillman on this somewhere)